cognite/neat/workflows/model.py

```diff
@@ -156,3 +156,29 @@ def float_to_int(cls, value):
         if isinstance(value, float):
             return int(value)
         return value
+
+
+class WorkflowConfigs(BaseModel):
+    """Workflow configs"""
+
+    configs: list[WorkflowConfigItem] = []
+
+    def get_config_item(self, config_name: str) -> WorkflowConfigItem:
+        return next((item for item in self.configs if item.name == config_name), None)
+
+    def set_config_item(self, config_item: WorkflowConfigItem):
+        for item in self.configs:
+            if item.name == config_item.name:
+                item.value = config_item.value
+                return
+        self.configs.append(config_item)
```

Comment:
I think it makes sense to just implement the `__get_item__` and `__set_item__` and treat the configs as a dict.